packages/main/src/themes/TabInOverflow.css

```diff
@@ -59,14 +59,11 @@
 	display: none;
 }
 
-.ui5-tab-container-responsive-popover [ui5-li-custom][focused]:first-child::part(native-li)::after {
-	border-top-left-radius: var(--_ui5_tc_overflowItem_focus_border);
-	border-top-right-radius: var(--_ui5_tc_overflowItem_focus_border);
-}
-
-.ui5-tab-container-responsive-popover [ui5-li-custom][focused]:last-child::part(native-li)::after {
-	border-bottom-left-radius: var(--_ui5_tc_overflowItem_focus_border);
-	border-bottom-right-radius: var(--_ui5_tc_overflowItem_focus_border);
+.ui5-tab-container-responsive-popover [ui5-li-custom][focused]::part(native-li)::after {
```

Comment:
Bring back the border-radius.
In Horizon, the focus should be rounded on all corners.
https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2698913922
In Quartz, the focus won't be rounded.

At first it won't look right because of the popover's different border-radius, but once the Popover's corners are updated with your other change, it should look better.

Reply:
No, the picture is not updated, in all popups where there are list the focus should NOT be rounded. This was deceided on last Thursdays meeting.